### DISCUSSION TITLE
Place/Location Records

### DISCUSSION THREAD

**Discussion by michael-wynne-wsu** [MukurtuCMS/Mukurtu-CMS#8] [Feature Enhancement]
Akin to person records (https://mukurtu.org/support/what-are-person-records-mukurtu-taxonomy-records/), we want to implement place/location records.

This could support aggregation of content (DH, Dictionary, etc) connected to a place, a selection of consistent fields to describe places, and free-text biography-style sections.

This function is wide open - what are useful fields? What kind of display is valuable?

**Comment by MonicaGalassi** [comment]
@michael-wynne-wsu this sounds like a great feature, and useful here in Aboriginal Countries for sure. I would say that display by map and language group would be of great interest. In this context, I think it would be useful to be able to align with existing thesaurus tools such as the [AIATSIS](https://thesaurus.aiatsis.gov.au/language/language.asp) one (keeping in mind that every community will use different spellings). DH media such as video and audio could be displayed on the map for showcasing oral histories. An easy integration with the homepage would be great too

**Reply by michael-wynne-wsu** [reply]
Monica, the big question I have about places is what descriptive information is needed about a place?

For example with the person records we have name, date of birth, date of death, related people (eg: family, mentors), and then a fairly free "biography" section.

For a place/location, I imagine we would want name and the "biography" or description type section, but are there any other other consistent fields that would be useful when discussing a place?

I do think viewing the aggregated content by map instead of just a list is a great idea.
